CMakeLists.txt

```diff
@@ -494,32 +436,33 @@ install(DIRECTORY tutorial
         PATTERN "*.png")
 
 # ---- Tools
-foreach(F mex_halide.m
-          GenGen.cpp
-          RunGen.h
-          RunGenMain.cpp
-          halide_benchmark.h
-          halide_image.h
-          halide_image_io.h
-          halide_image_info.h
-          halide_malloc_trace.h
-          halide_trace_config.h)
-  install(FILES "${HALIDE_BASE_DIR}/tools/${F}"
-          DESTINATION tools)
-endforeach()
+install(FILES
+        tools/mex_halide.m
+        tools/GenGen.cpp
+        tools/RunGen.h
+        tools/RunGenMain.cpp
+        tools/halide_benchmark.h
+        tools/halide_image.h
+        tools/halide_image_io.h
+        tools/halide_image_info.h
+        tools/halide_malloc_trace.h
+        tools/halide_trace_config.h
+        DESTINATION tools)
 
 # ---- README
-file(GLOB FILES "${HALIDE_BASE_DIR}/*.md")
-install(FILES ${FILES}
+install(FILES
+        CODE_OF_CONDUCT.md
+        README_cmake.md
+        README.md
+        README_rungen.md
+        README_webassembly.md
```

Comment:
We're going to forget to add new files to this list every single time.

Reply:
It might be fine to glob for non-source files, since it won't break incremental building. That said, the CMakeLists for src/ already use explicit file lists.

Reply:
I remember to do that because the Makefile uses an explicit list there too, and I've been disciplined about it. Better if we can enforce this automatically though.

Reply:
> We're going to forget to add new files to this list every single time.

Not if we all have the discipline to test with CMake as well before pushing :-)